### DISCUSSION TITLE
Roadmap of achoz version 0.1.0

### DISCUSSION THREAD

**Discussion by kcubeterm** [kcubeterm/achoz#11] [General]
When achoz will be release 0.1.0. following features should be available. 

Logger (Logs helps in figure out issue)
Test unit
PDF viewer
Video/audio preview

Expected date of releasing this version before 5 August 21

**Comment by cauerego** [comment]
i think 0.1.0 is already released, though. 😁

perhaps should call it 0.2.0...

or simply 202108xx+2 😏

i'll write another discussion for versioning.
